src/theme/components/index.ts

```diff
@@ -0,0 +1,15 @@
+import {
+  ButtonTheme,
+  ButtonOutlineTheme,
+  ButtonPrimaryTheme,
+  ButtonSecondaryTheme,
+  ButtonTransparentTheme,
+} from './Button'
+
+export default {
+  Button: ButtonTheme,
```

Comment:
don't need to required

Reply:
This is the default theme for Button (if we don't config in the project `extendTheme`), so I don't know what you mean by not required